src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -684,12 +684,28 @@ export default class StripViewModel extends BaseModel {
 
         this.insideCenter = insideCenter;
         this._transponder = transponderCode;
-        this._assignedAltitude = _round(assignedAltitude);
+        this._assignedAltitude = this._roundAltitude(assignedAltitude);
         this._flightPlanAltitude = flightPlanAltitude;
         this._arrivalAirport = arrivalAirportId;
         this._departureAirport = departureAirportId;
         this._flightPlan = flightPlan;
 
         return this._redraw();
     }
+
+    /**
+     * @for StripViewModel
+     * @method roundAltitude
+     * @param altitude
+     * @return roundedAltitude
+     */
+    _roundAltitude(altitude) {
+        const roundedAltitude = _round(altitude);
+
+        if (roundedAltitude === 'NaN') {
```

Comment:
`NaN` is a specific javascript object.  here, you are actually looking for the _string_ `NaN`. 
```javascript
'NaN' === NaN; // false
```

instead you should use lodash for this check (just because thats what we're using elsewhere):

```javascript
_isNaN(roundedAltitude)
```

question: what causes `_round(altitude)` to be `NaN`?

Reply:
I believe that, by default, the string '-' is set as altitude assigned, in the AircraftModel.

Reply:
thats what I thought 👍 

https://lodash.com/docs/4.17.4#isNaN